spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/repository/query/PartTreeDatastoreQuery.java

```diff
@@ -65,6 +68,8 @@
 	 * @param datastoreMappingContext used to provide metadata for mapping results to
 	 * objects.
 	 * @param entityType the result domain type.
+	 * Projection query. Regardless of this setting the method will be treated as a Projection query
```

Comment:
No longer applies.

Reply:
fixed. PR title also changed.

Reply:
@meltsufin  PTAL

Reply:
Hmmm, i thought I deleted those lines. I must've not pushed that commit. will fix.